deploy/tasks/pre-build.yaml

```diff
@@ -38,7 +35,7 @@ spec:
       mountPath: /var/workdir
   steps:
     - name: preprocessor
-      image: $(params.PNC_KONFLUX_TOOLING_IMAGE)
+      image: quay.io/konflux-ci/pnc-konflux-tooling:latest
```

Comment:
We should use a version tag instead of latest. IMO the control when to update (or roll-back) should be in the task definition rather than depending on the repository state.

Reply:
Konflux will update the tag to avoid me having to find the SHA

Reply:
Konflux pinning https://github.com/redhat-appstudio/jvm-build-service/pull/2315